consensus/hotstuff/cruisectl/block_time_controller.go

```diff
@@ -223,16 +223,47 @@ func (ctl *BlockTimeController) storeProposalTiming(proposalTiming ProposalTimin
 }
 
 // GetProposalTiming returns the controller's latest ProposalTiming. Concurrency safe.
-func (ctl *BlockTimeController) GetProposalTiming() ProposalTiming {
+func (ctl *BlockTimeController) getProposalTiming() ProposalTiming {
```

Comment:
made this method non-exported, because it wasn't used anywhere outside of the package.